packages/docs/src/pages/en/features/aliasing.md

```diff
@@ -31,13 +31,19 @@ export default createVuetify({
 })
 ```
 
+<alert type="info">
+
+Although treeshaking is automatically applied during production builds, it is advantageous to import components by specifying their full path in development mode. For instance, using 'vuetify/components/VBtn' instead of 'vuetify/components' ensures that the compiler loads fewer components, thus optimizing performance.
```

Comment:
```suggestion
Although treeshaking is automatically applied during production builds, it is advantageous to import components by specifying their full path in development mode. For instance, using `vuetify/components/VBtn` instead of `vuetify/components` ensures that the compiler loads fewer components, thus optimizing performance.
```